.jenkins/pytorch/test.sh

```diff
@@ -10,6 +10,7 @@ COMPACT_JOB_NAME="${BUILD_ENVIRONMENT}"
 source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
 
 echo "Testing pytorch"
+pip_install --user chainer
```

Comment:
Taking a dependency on chainer seems extremely odd. There must be a way to test what we want without it.